client-v2/src/shared/util/__tests__/validateImageSrc.spec.ts

```diff
@@ -190,7 +190,9 @@ describe('Validate images', () => {
       ).then(
         err => done.fail(err.toString()),
         err => {
-          expect(err.message).toMatch(/unable to locate/i);
+          expect(err.message).toMatch(
+            /There was a problem contacting The Grid/i
```

Comment:
I'm a little more explicit here, I hope that's OK.

Reply:
👍 This is much more useful